docs/src/main/paradox/routing-dsl/style-guide.md

```diff
@@ -0,0 +1,45 @@
+# Routing DSL style guide
+
+Akka HTTP's routing DSL is at the center of most Akka HTTP-based servers. It's where the incoming requests diverge into the different parts of the implemented services.
+
+Keeping all routing in one big structure will easily become hard to grasp and maintain. This page is tries to give a few hints for how you may want to break down the routing logic.
```

Comment:
`is tries` --> `tries` ?

Reply:
I would even remove the `tries` altogether:

```suggestion
Keeping all routing in one big structure will easily become hard to grasp and maintain. This page gives a few hints for how you may want to break down the routing logic.
```